### PR TITLE
tests: Deduplicate code logic in Jenkinsfile RSpec execution

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

**Comment by cpanato** [comment]
@mxinden can you please rebase when you have some time?
thanks for this work, make it much more easy to read :)

**Comment by mxinden** [comment]
@cpanato for sure. This is still a *Work in progess* PR. I will do so, once the other testing related PRs are merged. Really looking forward to this to make it into master.

**Comment by mxinden** [comment]
@cpanato test are failing due to current red master. Could you give this a review anyways?

**Comment by cpanato** [comment]
@mxinden this PR https://github.com/coreos/tectonic-installer/pull/2037 fixes the issues, can you please rebase?

**Comment by mxinden** [comment]
@coreosbot ok to test

File: tests/jenkins-jobs/tectonic_installer_public_pr_trigger_new.groovy [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  properties {
+    githubProjectUrl('https://github.com/coreos/tectonic-installer')
+  }
+
+  wrappers {
+    colorizeOutput()
+    timestamps()
+  }
+
+  // triggers {
```
Comment (cpanato) [inline tests/jenkins-jobs/tectonic_installer_public_pr_trigger_new.groovy]: please remove the comments when you think we can get it back )

**Comment by cpanato** [comment]
great job!! looks much more cleaner :)

**Comment by mxinden** [comment]
@cpanato Can you have another look? I dismissed your last review with some small changes.

**Pull request merged**